docs/user-guide/nextcloud-integration/README.md

```diff
@@ -61,7 +61,71 @@ The following video gives you a short overview of how to use this integration:
 
 ![OpenProject Nextcloud integration video](https://openproject-docs.s3.eu-central-1.amazonaws.com/videos/OpenProject-Nextcloud-Integration-2.mp4)
 
-Liking files to OpenProject work packages is currently only available via Nextcloud. First, in Nextcloud, navigate to the file or folder that you want to link to a work package and click on the *three dots → **Details**.*
+
+### In OpenProject
+
+In addition to listing files attached to a work package, the **Files** tab now also shows you Nextcloud files that are linked to the current work package. Hovering on any linked file with your mouse will give you options to  **open or download the file, show the containing folder in Nextcloud or remove the link**.
+![Empty_status_files](1_0_00-No_files_linked.png)
+
+![OP_linked_files](1_1_00-All_files_available.png)
+
+
+To link a Nextcloud file to the current work package, you can either:
+
+- select a file from your computer, which will be uploaded to Nextcloud and linked to this work package
+- select an existing file in Nextcloud to link to
+
+#### Upload and link file(s)
```

Comment:
@psatyal and @Kharonus is this ready? Will the code flag be removed with 12.5?

Reply:
Can we get a confirmation that this will be included in 12.5? @Kharonus